project/plugins.sbt

```
@@ -2,3 +2,4 @@ addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.11.0")
 addSbtPlugin("com.github.sbt" % "sbt-ci-release" % "1.5.12")
 addSbtPlugin("ch.epfl.scala" % "sbt-version-policy" % "2.1.3")
 addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.5.0")
+addSbtPlugin("com.eed3si9n" % "sbt-projectmatrix" % "0.9.0")
```

Latest sbt-projectmatrix is `0.9.1`

```suggestion
addSbtPlugin("com.eed3si9n" % "sbt-projectmatrix" % "0.9.1")
```